### ISSUE TITLE
Error when using `conda env create/update` (2.4.0)

### ISSUE THREAD

**Issue by koverholt** [conda/conda-env#156]
I upgraded to conda-env 2.4.0, but I get the following error when I run `conda env create` or `conda env update` on an environment file that worked with 2.3.0:

```
$ conda env create

Traceback (most recent call last):
  File "/Users/koverholt/anaconda/bin/conda-env", line 6, in <module>
    sys.exit(main())
  File "/Users/koverholt/anaconda/lib/python2.7/site-packages/conda_env/cli/main.py", line 68, in main
    return args_func(args, parser)
  File "/Users/koverholt/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 201, in args_func
    args.func(args, p)
  File "/Users/koverholt/anaconda/lib/python2.7/site-packages/conda_env/cli/main_create.py", line 77, in execute
    directory=os.getcwd())
  File "/Users/koverholt/anaconda/lib/python2.7/site-packages/conda_env/specs/__init__.py", line 18, in detect
    if spec.can_handle():
  File "/Users/koverholt/anaconda/lib/python2.7/site-packages/conda_env/specs/notebook.py", line 18, in can_handle
    self.nb = nbformat.reader.reads(open(self.name).read())
TypeError: coercing to Unicode: need string or buffer, NoneType found
```

**Comment by malev** [comment]
Fixed in `2.4.1`

**Comment by koverholt** [comment]
Thanks! Verified that this works in conda-env 2.4.1.
